#goal definition
nav_msgs/Path path
---
#result definition
---
#feedback
geometry_msgs/PoseStamped current_pose